api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
```

Comment:
Now after this command all the commands will get executed in the working directory as /app